src/cdk/testing/component-harness.ts

```diff
@@ -101,6 +101,8 @@ export interface HarnessLoader {
    * @return A list instances of the given harness type.
    */
   getAllHarnesses<T extends ComponentHarness>(query: HarnessQuery<T>): Promise<T[]>;
+
+  batchCD<T>(fn: () => Promise<T>): Promise<T>;
```

Comment:
I would do `batchChangeDetection` - I'm pretty strongly against using acronyms like this

(same for similarly named stuff)

Reply:
I used `batchCD` as a placeholder so we would remember to brainstorm names. I don't love how long `batchChangeDetection` is. However if we do add something like the `parallel` method mentioned in the PR description, I'd be fine with calling this `batchChangeDetection`.

Reply:
I kinda like `batchChangeDetection` because it's so straightforward. No way you can misinterpret a name like that